src/components/ReCaptcha/loader.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import RenderIf from '../RenderIf';
+import './styles.css';
+
+export default class ReCaptchaLoader extends Component {
+
+    componentDidMount() {
+        window.grecaptcha.ready(() => this.renderReCaptcha());
+    }
+
+    getContainerClassNames() {
+        const { className } = this.props;
+        return classnames('rainbow-recaptcha-container', className);
+    }
+
+    renderReCaptcha() {
+        const { value, theme, size, onChange } = this.props;
+        window.grecaptcha.render('reCaptcha-container', {
+            explicit: true,
+            sitekey: value,
+            theme,
+            size,
+            callback: onChange,
+            'expired-callback': onChange,
+            'error-callback': onChange,
+        });
+    }
+
+    render() {
+        const { error, style } = this.props;
+        return (
+            <div className={this.getContainerClassNames()} style={style}>
+                <div id="reCaptcha-container" />
+                <RenderIf isTrue={error}>
```

Comment:
Review the console and you will see a warning because you are not passing a bool here your are passing undefined, a node or a string. So:
```
<RenderIf isTrue={!!error}>
```